docs/core_docs/docs/expression_language/how_to/stream_log.mdx

```diff
@@ -0,0 +1,14 @@
+import CodeBlock from "@theme/CodeBlock";
+import Example from "@examples/guides/expression_language/stream_log.ts";
+
+# Stream Intermediate Steps
+
+All runnables also have a method `streamLog()` which is used to stream (as they happen) all or part of the intermediate steps of your chain/sequence.
+
+This is useful to show progress to the user, to use intermediate results, or to debug your chain.
+
+You can stream all steps (default) or include/exclude steps by name, tags or metadata.
+
+This method yields [JSONPatch](https://jsonpatch.com/) ops that when applied in the same order as received build up the RunState.
```

Comment:
Would add an example of how to actually reconstruct the JSON object too with a package